### PR TITLE
Quiz Submission

### PR DESCRIPTION
**Pull request by dmserrano**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+            <p><img class="img" src="https://encrypted-tbn2.gstatic.com/images?q=tbn:ANd9GcSQ74u44lb6CurgXGRLEeuQPpp95D0yGEkTGZU1NNL9GamDWI4F" alt="Product Image"/></p>
+            <p class="description">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut pharetra scelerisque pretium. Praesent fringilla nisi.</p>
+            <p class="available">
+              <input type="checkbox" checked disabled>Available</input>
+              <input type="checkbox" disabled>Not available</input>
+            </p>
+        </section>
+
+        <section>
+          <h3>Specifications</h3>
```
Comment (stevebrownlee) [inline index.html]: Missing `header` element

File: index.html [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+          <p class="bold">Size: 12oz Bottle</p>
+          <p class="bold">Weight: 0.2 lb</p>
+          <footer class="valid">Specifications valid until 02/15/17</footer>
+        </section>
+
+        <section>
+          <h3>Pricing</h3>
```
Comment (stevebrownlee) [inline index.html]: Missing `header` element

File: index.html [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          <p class="bold">Qty. 6: $8.49</p>
```
Comment (chortlehoort) [inline index.html]: No inline styles.  Use your CSS file to add style to HTML

File: quiz.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
+/*The entire card has a solid 1px border that is lightblue.
+The card title has a dotted 1px border that is lightgray.
+The title and product image are centered.
+Notice that the text for the description in the image is justified.
+The text for the product specification details and pricing details is bold.
+The availability element extends the full width of the card, with a dark grey background and yellow text.*/
+
+/*  base element selectorss  */
+
+* {
```
Comment (stevebrownlee) [inline quiz.css]: **Pro tip:** The `*` selector in CSS is a very expensive (i.e. memory intensive) operation for the browser.  Be as specific as you can possibly be with your CSS selectors.  You may want to consider using `body` here instead.

**Comment by stevebrownlee** [comment]
@DominicSerranoC14 Great job, Dominic.  Just a few minor issues to address.  When you're ready for me to review again, just send me a Slack message.

**Comment by stevebrownlee** [comment]
@DominicSerranoC14 Great job. Merge it!  👍

**Pull request merged**